src/js/step.js

```diff
@@ -247,16 +247,20 @@ export class Step extends Evented {
    * Triggers `destroy` event
    */
   destroy() {
-    if (isElement(this.el) && this.el.parentNode) {
-      this.el.parentNode.removeChild(this.el);
-      delete this.el;
-    }
-
     if (this.tooltip) {
       this.tooltip.destroy();
       this.tooltip = null;
     }
 
+    if (this.el && this.el instanceof HTMLElement) {
```

Comment:
This should be the same as it was before.
```suggestion
   if (isElement(this.el) && this.el.parentNode) {
```